documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -7,34 +11,62 @@ include::../_attributes.adoc[]
 :source-highlighter: highlight.js
 
 toc::[]
+endif::cdc-product[]
 
-Debezium's PostgreSQL Connector can monitor and record the row-level changes in the schemas of a PostgreSQL database.
+ifdef::cdc-product[]
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service-level agreements (SLAs) and might not be functionally complete; therefore, Red Hat does not recommend implementing any Technology Preview features in production environments. This Technology Preview feature provides early access to upcoming product innovations, enabling you to test functionality and provide feedback during the development process.
+For more information about support scope, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope^].
+====
+endif::cdc-product[]
+
+{ProductName}'s PostgreSQL Connector can monitor and record row-level changes in the schemas of a PostgreSQL database.
 
 The first time it connects to a PostgreSQL server/cluster, it reads a consistent snapshot of all of the schemas. When that snapshot is complete, the connector continuously streams the changes that were committed to PostgreSQL 9.6 or later and generates corresponding insert, update and delete events. All of the events for each table are recorded in a separate Kafka topic, where they can be easily consumed by applications and services.
 
+ifndef::cdc-product[]
 The connector has been tested and works with Postgres 9.6, 10, 11 and 12.
```

Comment:
@gunnarmorling I believe we can omit this since our release pages on the website provide this information per connector.  wdyt?

Reply:
Yes, good idea.